about/changelog.md

```diff
@@ -8,6 +8,15 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
+## 💡 Help with hypertables and faster notebooks
+<Label type="date">October 18, 2024</Label>
+
+### 🧙Hypertable creation wizard
+After creating a service, users can now create a hypertable directly in the Timescale Console by first creating a table, then converting it into a hypertable. This is possible through the in-console PopSQL SQL editors. All standard hypertable configuration options are supported, as is editing the table schema directly in the console to anything the user would like. 
+
+### 🍭 PopSQL Notebooks 
+The newest version of PopSQL Notebooks is now waaaay faster.  Why? We've incorporated the newly developed v3 of our query engine that currently powers the Timescale Cloud console SQL Editor.  Check out the difference in query response times.
```

Comment:
Are you saying:
- PopSQL (external web tool) Notebooks are faster because they now use the same query engine as SQL Editor
- Notebooks in Data mode (PopSQL in Timescale Console) are faster because it now uses the same query engine as SQL Editor.

SQL Editor is in Timescale Console, not Timescale Cloud console

Reply:
I'm actually not sure what Justin is referring to, Data mode or the standalone PopSQL.  I guess they are probably the same now.  I'll update
I thought we were referring to it as Timescale Cloud now?

Reply:
Timescale Cloud is the whole shebang. 
Timescale Console is the UI you use to administer Timescale Cloud and interact with data in the individual services.